### ISSUE TITLE
a11y: Color Contrast - API Key

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13731]
Insufficient color contrast for standard text

## Actual Result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3 
Background Text Color: #FFFFFF
Ratio: 3.16:1
Main Content
- API Key

Foreground Text Color: #3D98D3
Background Text Color: #E2F0F8
Ratio: 2.72:1
Main Content
- Copy button (in default state)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3 
Ratio: 3.16:1
Main Content
- Done button (in default state)"

### Create API Key

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1
Main Content
- API Key (in default state)
- Cancel button (in default state)
- Description (in typing state)
- Scope (Dropdown on keyboard focus)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1
Main Content
- Create button (by default state) 
- No Scope, local, vpat-testing-downstream, Hours, Days and Month (dropdown option on hover and keyboard focus)

Foreground Text Color: #FFFFFF 
Background Text Color: #B6B6C2
Ratio: 2.0:1
Main Content
- No Scope, local, vpat-testing-downstream, Hours, Days and Month (dropdown option in selected state when keyboard focus is not on option)

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF
Ratio: 1.7:1
Main Content
- Optionally enter a description to help you identify this API Key

Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.1:1
Create Project
Main Content
- Project (in default state)
- Name (in typing state)
- Description (in typing state)
- Cancel button (in default state)

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF
Ratio: 1.7:1
Create Project
Main Content
- A unique name
- Any text you want that better describes this resource

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.1:1
Create Project
Main Content
- Create button (by default state) "

"Create API
Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content 
- 90 days - Maximum allowed and Custom (unselected radio button)
- Duration input field (available after selecting the Custom radio button)  

Delete Project
Component
- Enter c-m-tcmmzqxw below to confirm input field

DaemonSet: Create
- Init Container, Standard Container, None, Add command to execute and Create HTTP request.  (unselected radio buttons) 

StorageClass: Create
- GP2 - General Purpose SSD, GP3 - General Purpose SSD, IO1 - Provisioned IOPS SSD, ST1 - Throughput-Optimized HDD and so on  (unselected radio buttons) 

Repository: Create
Main Content 
- http(s) URL to an index generated by Helm,Git repository containing Helm chart or cluster template definitions, OCI Repository  (unselected radio buttons) 

Banners
Main Content 
 - Left, Center, Right unselected radio button "

### Delete API Key

"Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio: 3.16:1
Main Content 
- Cancel button (in default state) 

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.1:1
Main Content 
- Cancel and Delete buttons (on focus) 

Foreground color: #FFFFFF
Background color: #F64747
Contrast ratio: 3.5:1
Main Content 
- Delete button (in default state)

Foreground color: #FFFFFF
Background color: #F41616
Contrast ratio: 4.2:1
Main Content 
- Delete button (on hover)

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.1:1
Delete Project
Component
- Cancel button (in default state)

Foreground color: #FFFFFF
Background color: #F64747
Contrast ratio: 3.5:1
Delete Project
Component
- Delete button (in default state when user enter the data to confirm)"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground color: #2F68DF
> Background color: #EDF2FC
> Contrast ratio: 4.4:1
> - Cancel (On hover on focus )
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 107
